README.md

```diff
@@ -11,21 +11,40 @@
 ### Usage
 
 ```
-$ jake --help
-usage: jake [-h] [-S] [-P] [-V] [-VV] [-A APPLICATION] [-C] {ddt}
-
-positional arguments:
-  {ddt}                 run jake
-
-optional arguments:
-  -h, --help            show this help message and exit
-  -S, --snake           set optional jake config
-  -P, --python          set optional jake IQ Server config
-  -V, --version         show program version and exit
-  -VV, --verbose        set verbosity level to debug
-  -A APPLICATION, --application APPLICATION
-                        supply an IQ Server Public Application ID
-  -C, --clean           wipe out jake cache
+$ Usage: jake [OPTIONS] COMMAND [ARGS]...
+
+  Jake: Put your python deps in a chokehold.
+
+Options:
+  -v, --version  Print version and exit
+  --help         Show this message and exit.
+
+Commands:
+  config  Allows a user to set Nexus IQ or OSS Index config params...
+  ddt     SPECIAL MOVE Allows you to perform scans backed by Sonatype's OSS...
+  iq      EXTRA SPECIAL MOVE Allows you to perform scans backed by...
+
+
+$ Usage: jake ddt [OPTIONS]
+
+  SPECIAL MOVE
+
+  Allows you to perform scans backed by Sonatype's OSS Index
+
+  Example usage:
+
+      Python scan: jake ddt
+
+      Conda scan: conda list | jake ddt -c
+
+      Clear cache: jake ddt --clear
+
+Options:
+  -vv, --verbose  Set log level to verbose
```

Comment:
I think this "multi-character" shortcut patter is gonna cause trouble for POSIX command parses (like Cobra). We should probably standardize on "--vv", e.g. prefix with two dashes.